src/components/repository-browser.js

```diff
@@ -106,6 +95,22 @@ class RepositoryBrowser extends React.Component {
         sorter: firstBy('lastModified')
       }
     ];
+
+    if (hasUserPermission(this.props.user, permissions.DELETE_CDN_FILE)) {
+      this.columns.unshift({
+        dataIndex: 'displayName',
+        key: 'displayName',
+        render: this.renderDeleteColumn,
```

Comment:
Could we maybe only render the column if `!isDirectory`?

When I tested deleting a directory it did not work, but I guess we don't want "directory" deletion, or would be safer not to allow it?

Reply:
We don't want that, I missed that, I'll try to fix it, thanks!

Reply:
This is fixed in the renderer, it's not really possible here because you don't have yet the knowledge if the row is a directory or not.

Reply:
Oh of course in the renderer, I didn't imply to fix it **exactly here** 😊  I mentioned "only render the column if !isDirectory", since this is the place that sets up the column.